rl/agents/sarsa.py

```diff
@@ -0,0 +1,196 @@
+from rl.core import Agent
+import warnings
+from copy import deepcopy
+
+import numpy as np
+from keras.callbacks import History
+
+from rl.callbacks import TestLogger, TrainEpisodeLogger, TrainIntervalLogger, Visualizer, CallbackList
+from rl.agents.dqn import mean_q
+from rl.util import huber_loss
+from keras.layers import Input, Lambda
+from keras.models import Model
+import keras.backend as K
+from rl.policy import EpsGreedyQPolicy
+from rl.util import get_object_config
+
+
+class Sarsa(Agent):
+    def __init__(self, model, nb_actions, policy=EpsGreedyQPolicy(), gamma=.99, nb_steps_warmup=10,
+                 train_interval=1, delta_range=None, delta_clip=np.inf, *args, **kwargs):
```

Comment:
Can you please remove `delta_range`? It is still part of `DQNAgent` but deprecated and we don't need to keep it around for this new agent.